JASP-Engine/JASP/R/robustbayesianmetaanalysis.R

```diff
@@ -1488,13 +1513,13 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
     }
     
     # deal with JASPgraphs screwing secondary axis label distance
-    if(! (parameters == "theta" || (parameters == "omega" && options[["plots_omega_function"]]) ) ){
-      if(p$plot_env$any_density && p$plot_env$any_spikes){
-        p <- p + ggplot2::theme(
-          axis.title.y.right = ggplot2::element_text(vjust = 3),
-          plot.margin = ggplot2::margin(t = 3, r = 12, b = 0, l = 1))
-      }
-    }
+    if (!all(parameters %in% c("mu", "tau")))
+      if (! (parameters == "theta" || (parameters == "omega" && options[["plots_omega_function"]]) ) )
+        if(p$plot_env$any_density && p$plot_env$any_spikes)
```

Comment:
This line looks a bit like black magic to me. Perhaps this should be part of the fit object?

Reply:
Yeah, I agree that this is not a very nice solution, however, the fit object does not need to know whether the model-averaged posterior distribution is a mixture of spikes or densities or not, especially since it varies for each parameter. Therefore, the computations are done internally in the plotting function. I think that the problem is on side of `themeJasp()` because it doesn't space the secondary y-axis from the figure in the same way as it does with the primary y-axis.